git-merge-dep-updates.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
```

Comment:
a) Add some explanatory comment / documentation for this command.
b) We'll want this everywhere dependabot runs, right? Is there an easy way to avoid maintaining the same script in 3 repos? e.g., make this script always available on dev1 via a different mechanism than git, or set it up separately as a submodule, or something else?

Reply:
re: b, that sounds good if:
* We still check the file into version control somewhere (maybe https://github.com/b12io/crowdsurfing/tree/master/infrastructure/scripts ?)
* You place it on the path for both dev1 and dev2 after the PR is merged.
* We update the docs for creating new devN machine so we remember to copy it over in the future (or is there a script to which we can add a `cp` command so it happens automatically?)